src/components/Undo/Undo.js

```diff
@@ -0,0 +1,59 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const actions = require("actions/index");
+const gBlockedLinks = require("lib/blockedLinks");
+
+const Undo = React.createClass({
+  /**
+   * The undo dialog's timeout in miliseconds.
+   */
+  HIDE_TIMEOUT_MS: 15000,
+  timeout: null,
```

Comment:
Note that you could safely set this to 0 on initialization. The HTML spec mandates that `setTimeout()` always returns a value greater than 0, so no need to initialize as null.